uptane-standard.md

```diff
@@ -449,14 +457,14 @@ The following sections describe the role-specific metadata. All roles SHALL foll
 
 A repository's Root metadata distributes the public keys of the top-level Root, Targets, Snapshot, and Timestamp roles, as well as revocations of those keys. It SHALL contain two attributes:
 
-* A representation of the public keys for all four roles. Each key should have a unique public key identifier.
+* A representation of the public keys for all four roles. Each key shall have a unique public key identifier.
```

Comment:
Good catch, but then `SHALL` should be capitalized.

Reply:
done.